lib/DatabaseLayer.py

```diff
@@ -216,10 +216,10 @@ def getCVEs(limit=False, query=[], skip=0, cves=None, collection=None):
     cve=col.find(query[0]).sort("Modified", -1).limit(limit).skip(skip)
   else:
     cve=col.find({"$and": query}).sort("Modified", -1).limit(limit).skip(skip)
-  return sanitize(cve)
+  return {'cves': sanitize(cve), 'total': cve.count()}
```

Comment:
I am not entirely sure that this is needed. You use `total` only in jinja code, but the count could be obtained using [builtins](https://jinja.palletsprojects.com/en/2.10.x/templates/#list-of-builtin-filters), could it not?

Reply:
in this case the count() function is db.collection.count() from mongo that returns the total number of results from the query that is required to determine the "last page" of the pagination
- https://docs.mongodb.com/manual/reference/method/db.collection.count/

the getCVEs function however returns just a subset of these results (determined by limit and skip), so later in code you are not able to determine the total number required for pagination